product_docs/docs/tde/15/enabling/enabling_tde.mdx

```diff
@@ -21,12 +21,12 @@ This example uses EDB Postgres Advanced Server 15 running on a Linux platform. I
    export PGDATAKEYUNWRAPCMD='openssl enc -d -aes-128-cbc -pass pass:ok -in %p'
    ```
    !!!note
-      - If you are on Windows you don't need the single quotes around the variable value.
+      - If you're on Windows, you don't need the single quotes around the variable value.
       
       - Ensure you replace `ok` with the passphrase you want to use to wrap the data encryption key.
```

Comment:
If that's the case, should ok be written as <ok>? If so, I would take the second bullet out of the note and add a regular sentence after the code: Replace <ok> with the passphrase you want to use to wrap the data encryption key.

OR, perhaps you want to change ok to <passphrase>?
Check for all instances of ok if you agree a change should be made. (Or I will...)

Reply:
@gvasquezvargas  that was supposed to indicate variable (less than greater than around ok). Is that what you mean to do? Do you understand what I mean? If ok is a variable, it should be enclosed in angle brackets, and the text that follows should just say what to replace it with, then it doesn't need to be in the note. LMK if you need more clarification on the query

Reply:
In the example command, by using the wording `ok`, the user is actually setting the password to "ok". In other words, their passphrase/password to access the encrypted files is literally "ok".

I think it is a good idea to enclose it with angle bracket to denote that the use must provide a value, so we could replace it with `<password>`. And then we can remove the notes as it would be self-explanatory. If you do so, could you check all instances in examples, please 🙏 

TY!

Reply:
Yes, that's a good idea. I'll take care of it.